beets/library.py

```diff
@@ -1648,6 +1648,20 @@ def tmpl_ifdef(self, field, trueval=u'', falseval=u''):
             return falseval
 
 
+def apply_item_changes(lib, item, move, pretend, write):
+    """Store, move and write the item according to the arguments.
+    """
```

Comment:
This function seems useful, but it's a little out of place—for one thing, it's in between the definition of the default template functions and the little computation that gathers them up into a list. It's also mostly relevant to plugin actions, where this whole "pretend" notion has meaning, for example. What do you think about moving this to the `plugins` module?

This docstring could also use a little expansion to explain what "move" and "pretend" do, for example. I realize that you didn't write this docstring, but now might be a good opportunity to improve it. :smiley:

Reply:
Good call! Docstring improved and moved to `plugins.` I was also considering moving it to the `autotag` module alongside `apply_item_metadata` and `apply_metadata`, although not sure what makes more sense.